features/mbedtls/inc/mbedtls/config.h

```diff
@@ -2613,4 +2613,15 @@
 #include "check_config.h"
 
 #endif /* !MBEDTLS_ENTROPY_HARDWARE_ALT && !MBEDTLS_TEST_NULL_ENTROPY */
+
+#if defined(MBEDTLS_TEST_NULL_ENTROPY)
```

Comment:
Janos commented that perhaps we should include NV_SEED in these preprocessor conditions as well.